src/triggers/__tests__/createPullRequestForJiraIssue.test.ts

```diff
@@ -146,12 +146,12 @@ describe('createPullRequestForJiraIssue', () => {
     jiraIssueSpy.mockImplementation((_key: string) =>
       Promise.resolve(noRepository)
     )
-    await createPullRequestForJiraIssue(email, issueKey)
+    await createPullRequestForJiraIssue(email, '')
```

Comment:
@DanielM-SR does this test need to change? If we don't pass a repository here as part of `issueKey ` then I think this test should still pass the way it was?

Reply:
Ah... I see why I had to change that originally. It was failing ,but that was before I added the `mockJiraIssue` part back into the test. It passes now so I will change it back. 👍🏽